it/mail/chrome/messenger/addressbook/addressBook.properties

```diff
@@ -103,75 +103,75 @@ stateZipSeparator=
 prefixTo=A
 prefixCc=CC
 prefixBcc=Ccn
-emptyEmailCard=Please remove the following cards with no email address : 
+emptyEmailCard=Si prega di rimuovere i seguenti contatti che non presentano indirizzi e-mail:
```

Comment:
CORRECTION NEEDED: I would shorten a bit the sentence as follows: 

 +emptyEmailCard=Si prega di rimuovere i seguenti contatti sprovvisti di indirizzo e-mail: